hack/test-e2e.sh

```diff
@@ -32,21 +38,30 @@ https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prom
 oc create -n ${NAMESPACE} -f \
 https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/servicemonitor.crd.yaml || :
 
-repo_dir="$(dirname $0)/.."
-$repo_dir/hack/gen-olm-artifacts.sh ${CSV_FILE} ${NAMESPACE} 'ns' | oc create -f - || :
+repo_dir=$(echo "$(cd "$(dirname "$1")"; pwd -P)")
```

Comment:
@richm not sure if i should revert this.  I was seeing strange output trying to deploy the EO and then reverted that logic when I realized the e2e test doesn't need to deploy the EO to do a test on the CLO and what it does.